client/CServerHandler.cpp

```diff
@@ -243,7 +245,7 @@ void CServerHandler::startLocalServerAndConnect()
 #else
 	const ui16 port = 0;
 #endif
-	justConnectToServer(settings["server"]["server"].String(), port);
+	justConnectToServer(localhostAddress, port);
```

Comment:
Why localhost? Where do we read settings["server"]["server"] now?

Reply:
Local games?

Reply:
yes, check the method name ;)